### PR TITLE
merchant map request and directions added

### PR DESCRIPTION
**Pull request by islandbitcoin**

### CODE REVIEW

File: app/graphql/generated copy.gql
Comment (Nodirbek75) [inline app/graphql/generated copy.gql:1]: Need to delete if not used

File: app/graphql/generated.gql
Comment (Nodirbek75) [inline app/graphql/generated.gql:1]: Need to delete if not used

File: app/graphql/local-schema copy.gql
Comment (Nodirbek75) [inline app/graphql/local-schema copy.gql:1]: Need to delete if not used

File: app/graphql/generated copy.ts
Comment (Nodirbek75) [inline app/graphql/generated copy.ts:1]: Need to delete if not used

**Comment by islandbitcoin** [comment]
updated, but still would like to explore the async storage of blink pins. For now I'm using "cache-only" so we dont pull from blink.

**Review by Nodirbek75 — APPROVED** [review]

**Pull request merged**